src/Tests/Microsoft.Diagnostics.Monitoring.Tool.UnitTests/ProcessTests.cs

```diff
@@ -245,7 +245,15 @@ private void VerifyProcessInfoEquality(List<ProcessInfo> processInfos)
 
             Assert.Single(processInfoPIDs.Distinct());
             Assert.Single(processInfoUIDs.Distinct());
-            Assert.Single(processInfoNames.Distinct(StringComparer.Ordinal));
+            // In .NET 5+, the process name comes from the command line from the ProcessInfo command, which can fail
+            // or be abandoned if it takes too long to response. In .NET Core 3.1, the process name comes from the
```

Comment:
Just a typo: response -> respond

Reply:
Thanks! I've been doing too much HTTP and IPC work recently (lots of "response" everywhere).